src/apps/frontend/components/form-input/form-input.component.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { FormControl as BaseFormControl, FormControlProps } from 'baseui/form-control';
+import React from 'react';
+
+const FormControl: React.FC<React.PropsWithChildren<FormControlProps>> = ({
+  children,
+  ...props
+}) => (
+  <BaseFormControl
+    {...props}
+    overrides={{
+      Label: {
+        style: ({ $theme }) => ({
+          color: `${$theme.colors.fontColor}`,
```

Comment:
Why do we need to override this? Can we not and use the form control directly at consumer?

Reply:
Done